proteuslib/unit_models/reverse_osmosis_0D.py

```diff
@@ -275,29 +276,81 @@ def build(self):
             units=units_meta('mass')*units_meta('length')**-3,
             doc='Pure water density')
 
-
         # Add unit variables
+        flux_mass_io_phase_comp_initialize = {}
+        for t in self.flowsheet().config.time:
+            for io in self.io_list:
+                for j in self.config.property_package.component_list:
+                    if j in self.solvent_list:
+                        flux_mass_io_phase_comp_initialize[(t, io, 'Liq', j)] = 1e-3
+                    elif j in self.solute_list:
+                        flux_mass_io_phase_comp_initialize[(t, io, 'Liq', j)] = 1e-6
```

Comment:
It shouldn't hold up this PR, but to me it's more in the Pyomo-style to pass a function to initialize, which unwinds these for loops behind the scenes:
```
def flux_mass_io_phase_comp_initialize(b, t, io, j):
    if j in self.solvent_list:
        return 1e-3
    elif j in self.solute_list:
        return 1e-6
```
There should maybe be an "else" clause in this function, depending on how defensively we're programming.

Reply:
Yeah I'll update this. There is already a check for if component j is not a solvent or a solute and it raises an Exception, so the else clause is probably unnecessary.